pkg/kclient/kclient.go

```
@@ -606,6 +606,54 @@ func (c *Client) GetPodLogs(pod *corev1.Pod, file *os.File) (err error) {
 	return err
 }
 
+// ExecPodCmd executes command in the pod container
+func (c *Client) ExecPodCmd(command []string, containerName, podName string) (string, string, error) {
+
+	fmt.Printf("Executing command: %s in pod: %s container: %s\n", strings.Join(command, " "), podName, containerName)
+
+	clientset := c.KubeClient
+	config := c.KubeClientConfig
+	namespace := c.Namespace
+
+	req := clientset.CoreV1().RESTClient().Post().
+		Resource("pods").
+		Name(podName).
+		Namespace(namespace).
+		SubResource("exec")
+	scheme := runtime.NewScheme()
+	if err := corev1.AddToScheme(scheme); err != nil {
+		panic(err)
+	}
+
+	parameterCodec := runtime.NewParameterCodec(scheme)
+	req.VersionedParams(&corev1.PodExecOptions{
+		Command:   command,
+		Container: containerName,
+		Stdin:     false,
+		Stdout:    true,
+		Stderr:    true,
+		TTY:       false,
+	}, parameterCodec)
+
+	exec, err := remotecommand.NewSPDYExecutor(config, "POST", req.URL())
+	if err != nil {
+		panic(err)
+	}
+
+	var stdout, stderr bytes.Buffer
+	err = exec.Stream(remotecommand.StreamOptions{
+		Stdin:  nil,
+		Stdout: &stdout,
+		Stderr: &stderr,
+		Tty:    false,
+	})
+	if err != nil {
+		panic(err)
```

Can we return the error instead of the panic?